src/tests/unit/tests/reports/package/__snapshots__/integration.test.ts.snap

```diff
@@ -117,7 +117,7 @@ exports[`report package integration with issues 1`] = `
             color-interpolation-filters="sRGB"
             filterUnits="userSpaceOnUse"
             height="29.5269"
-            id="filter0_d"
+            id="filter000_d"
```

Comment:
Similar to my other comment about ms-grid related classes, this is a side effect from the office fabric class name normalizer update. Unfortunate but I think is not too bad.